gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/shift/SweNetworkShifter.java

```diff
@@ -94,9 +93,7 @@ public void shiftNetwork(double stepValue, Network network) throws GlskLimitatio
                     double asked = entry.getValue();
                     String logApplyingVariationOnZone = String.format("[%s] : Applying variation on zone %s (target: %.2f)", direction, zoneId, asked);
                     LOGGER.info(logApplyingVariationOnZone);
-                    ScalingParameters iterativeScalingParameters = new ScalingParameters();
-                    iterativeScalingParameters.setIterative(true);
-                    double done = zonalScalable.getData(zoneId).scale(network, asked, iterativeScalingParameters);
+                    double done = zonalScalable.getData(zoneId).scale(network, asked);
```

Comment:
If the iterative parameter is removed from there, it should be added somewhere else

Reply:
More precisely, it should be added in the ZonalScalableProvider class (l.68): `return Scalable.proportional(percentages, scalables, true);` (it is the `true`parameter that indicates the "iterativeness")